<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCUProyecto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create and Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReadProyecto">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Consulta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCUActividades">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create and Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReadActividades">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bienvenido</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Proyecto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actividades</string>
    </property>
   </widget>
  </widget>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionEmpleados">
   <property name="text">
    <string>Empleados</string>
   </property>
  </action>
  <action name="actionUsuarios">
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="actionAjustes">
   <property name="text">
    <string>Ajustes</string>
   </property>
  </action>
  <action name="actionCerrar_Session">
   <property name="text">
    <string>Cerrar Sesion</string>
   </property>
  </action>
  <action name="actionClientes">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionProveedores">
   <property name="text">
    <string>Proveedores</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
